docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How to register a broker
```

Comment:
We don't do "How to" titles, do we? 

"Register a ClusterServiceBroker" -> this matches the actual content of this GSG.

Reply:
Good point, I'll change the title to: "Register a broker in the Service Catalog"